Yoast/YoastSEO.js
# Remove unused morphology files

## Summary

This PR can be summarized in the following changelog entry:

* 

## Relevant technical choices:

* Deletes the unused morphology files that have been transported to one data-file.

## Test instructions

This PR can be tested by following these steps:

* Check if all tests still run.
* Check if the morphology functionality still works (e.g., the IntroductionKeyword assessment)

Fixes #1751
